Cargo.toml

```diff
@@ -187,6 +187,7 @@ http-range-client = { git = "https://github.com/pka/http-range-client.git", defa
   "reqwest-sync",
   "rustls-tls",
 ] }
+sevenz-rust = "0.6.1"
```

Comment:
There is also [sevenz-rust2](https://lib.rs/crates/sevenz-rust2), which seems to be maintained.